examples/cifar10_resnet.py

```diff
@@ -135,29 +135,26 @@ def resnet_block(inputs,
     # Returns
         x (tensor): tensor as input to the next layer
     """
+    conv2D = Conv2D(num_filters,
```

Comment:
By any chance can we use a better naming convention instead of having the same spelling but different case for the alias?

Reply:
@ozabluda : I agree that the current `resnet_block` should actually be called `resnet_layer` since it is only a layer in the resnet block. The current `sub_block` should actually be the `resnet_block`. We should use `resnet_block` instead of residual unit to be consistent with other architectures.